perl/colorize_regex.pl

```diff
@@ -0,0 +1,1396 @@
+# SPDX-FileCopyrightText: 2025 ryoskzypu <ryoskzypu@proton.me>
+#
+# SPDX-License-Identifier: MIT-0
+#
+# License:
+#   MIT No Attribution
+#
+#   Copyright (c) 2025 ryoskzypu <ryoskzypu@proton.me>
```

Comment:
This copyright line should be removed (duplicate with the SPDX copyright tag above) as it would generate a duplicate copyright line in the SPDX bill of materials:

```
$ reuse spdx

(...)

FileName: ./perl/colorize_regex.pl
SPDXID: SPDXRef-f61225f4475537305dbb661a5ed0d658
FileChecksum: SHA1: cc0594b6b3513c733fbc41316dc0007716ba94dd
LicenseConcluded: NOASSERTION
LicenseInfoInFile: MIT-0
FileCopyrightText: <text>Copyright (c) 2025 ryoskzypu <ryoskzypu@proton.me>
SPDX-FileCopyrightText: 2025 ryoskzypu <ryoskzypu@proton.me></text>

(...)
```